File: WebApp/Models/PH/PH_AIRST.cs
using System;
using System.Collections.Generic;

namespace WebApp.Models
{
    public class PH_AIRST : JCLib.Mvc.BaseModel
    {
        public string AGEN_NO { get; set; }
        public string FBNO { get; set; }
        public string NAMEC { get; set; }
        public int SEQ { get; set; }
        public DateTime TXTDAY { get; set; }
        public string AIR { get; set; }
        public string DEPT { get; set; }
        public string XSIZE { get; set; }
    }
}

File: WebAppVen/Scripts/app/model/PH_AIRST.js
Ext.define('WEBAPP.model.PH_AIRST', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'AGEN_NO', type: 'string' },
        { name: 'FBNO', type: 'string' },
        { name: 'MMCODE', type: 'string' },
        { name: 'SEQ', type: 'int' },
        { name: 'TXTDAY', type: 'date' },
        { name: 'AIR', type: 'string' },
        { name: 'DEPT', type: 'string' },
        { name: 'XSIZE', type: 'string' },

        { name: 'CREATE_TIME', type: 'date' },
        { name: 'CREATE_USER', type: 'string' },
        { name: 'EXTYPE', type: 'string' },
        { name: 'STATUS', type: 'string' },
        { name: 'UPDATE_TIME', type: 'date' }
    ]
});